starboard/nplb/media_buffer_test.cc

```diff
@@ -266,13 +278,25 @@ TEST(SbMediaBufferTest, MaxCapacity) {
 }
 
 TEST(SbMediaBufferTest, Padding) {
+#if SB_API_VERSION >= 16
+  // SbMediaGetBufferPadding() was deprecated in Starboard 16, its return value
+  // is no longer used when allocating media buffers.  This is verified
+  // explicitly here by ensuring its return value is 0.
+  // An implementation that has specific padding requirement should make a
+  // copy of the incoming buffer when necessary.
+  EXPECT_EQ(SbMediaGetBufferPadding(), 0);
```

Comment:
Add a check in `starboard/media.h` to guard SbMediaGetBufferPadding?

#if SB_API_VERSION < 16

Reply:
I am trying to keep the function in SB16 so the nplb test will fail explicitly for an existing implementation returning a non-supported alignment (say 16).  Otherwise if the platform maintainer misses the entry in the change log and keeps using the existing implementation, playback may still work in most cases but fail randomly.